docs/resources/shadow.md.erb

```diff
@@ -89,56 +89,58 @@ The `count` matcher tests the number of times the named user appears in `/etc/sh
 
 This matcher is best used in conjunction with filters. For example:
 
-    describe shadow.users('dannos') do
+    describe shadow.user('dannos') do
        its('count') { should eq 1 }
     end
 
-### expiry_date
-
-The `expiry_date` matcher tests the number of days a user account has been disabled:
-
-    its('expiry_date') { should eq '' }
+### user
 
-### inactive_days
+The `user` matcher tests if the user name exists `/etc/shadow`:
 
-The `inactive_days` matcher tests the number of days a user must be inactive before the user account is disabled:
+    its('user') { should eq 'root' }
 
-    its('inactive_days') { should eq '' }
+### password
 
-### last_changes
+The `password` matcher returns the encrypted password string from the shadow file. The returned string may not be an encrypted password, but rather a `*` or similar which indicates that direct logins are not allowed.
 
-The `last_changes` matcher tests the last time a password was changed:
+For example:
 
-    its('last_changes') { should eq '' }
+    its('password') { should cmp '*' }
 
-### max_days
+### last_change
 
-The `max_days` matcher tests the maximum number of days after which a password must be changed:
+The `last_change` matcher tests the last time a password was changed:
 
-    its('max_days') { should eq 90 }
+    its('last_change') { should eq '' }
```

Comment:
Will `last_change` ever be `''`? Or, will it be days since the user was created and the password was set?

Reply:
Its generally going to be a timestamp (epoch since last change), but its entirely possible to be empty too. 

Eg, system users e.g. `bin` or `daemon` may not have a password set, or any values for the other fields. Its entirely dependent on the `useradd` program doing the right thing and referencing `login.defs`. Its also possible to update the shadow file without `useradd` so anything can happen really.